expression/util.go

```diff
@@ -928,19 +929,27 @@ func MaybeOverOptimized4PlanCache(ctx sessionctx.Context, exprs []Expression) bo
 		RemoveMutableConst(ctx, exprs)
 		return false
 	}
-	return containMutableConst(ctx, exprs)
+	return containMutableConst(exprs)
+}
+
+func maybeOverOptimized4PlanCacheWithStmtCtx(sc *stmtctx.StatementContext, exprs []Expression) bool {
```

Comment:
After merging https://github.com/pingcap/tidb/pull/29782, this function can be removed, is it right?